utils/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/sink/KafkaAvroSinkFactoryImpl.scala

```diff
@@ -0,0 +1,22 @@
+package pl.touk.nussknacker.engine.avro.sink
+
+import pl.touk.nussknacker.engine.api.LazyParameter
+import pl.touk.nussknacker.engine.api.process.Sink
+import pl.touk.nussknacker.engine.avro.RuntimeSchemaData
+import pl.touk.nussknacker.engine.avro.encode.ValidationMode
+import pl.touk.nussknacker.engine.kafka.serialization.KafkaSerializationSchema
+import pl.touk.nussknacker.engine.kafka.{KafkaConfig, PreparedKafkaTopic}
+import pl.touk.nussknacker.engine.util.KeyedValue
+
+trait KafkaAvroSinkFactoryImpl {
```

Comment:
I think it's more KafkaAvroSinkImplementationFactory :) (also in other places)

Reply:
good point - fixed